src/components/pages/maintenance/maintenance.scss

```diff
@@ -15,6 +15,13 @@
     @include rem-font-size(48px);
   }
 
+  .maintenance-sub-header {
```

Comment:
Does the maintenance page need to updated in this PR?
I'd recommend removing this change into a separate PR.